macros/sfdc_get_formula_column_values.sql

```diff
@@ -27,7 +27,13 @@
                         end as value
             from {{ target_relation }}
             where object = '{{ join_to_table }}'    -- This filters the query to only include the formula fields referenced by the source table.
+
+            {% if added_inclusion_fields is not none %} -- If a users has designated fields_to_include then those will be the only fields included in the macro.
+                and {{ key }} in {{ added_inclusion_fields }}
+            {% else %}
                 and {{ key }} not in {{ var('sfdc_exclude_formulas',"('')") }} -- Excludes any formula fields the user may have set within the sfdc_exclude_formulas variable. Default is an empty string.
```

Comment:
is there a case in which someone would want the same sort of global/non-source-table-specific scope of `sfdc_exclude_formulas` but for including formulas??

Reply:
I thought a lot about this scenario and have coordinated with Ray that this is the avenue we want to go for inclusion instead of the global variable.

The main reason is we want users to be intentional about the fields they want to include for each model. If they do a global variable then there is a lot that can get lost across models. We decided to keep the exclude global for now simply for error handling rather than intentional desire to exclude.